ROOTFS/etc/waggle/sanity/fatal/core_sdcard.test

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+pf="Recovery SDCard test"
+echo "$pf: Begin"
+
+# run fsck on our sd card
+fsck -n -f /dev/mmcblk1p1
+
+if [ "$?" != "0" ];
+then
+        echo "$pf: /dev/mmcblk1p1 failed fsck FAIL"
+        exit 1
+fi
+
+fsck -n -f /dev/mmcblk1p2
+
+if [ "$?" != "0" ];
+then
+        echo "$pf: /dev/mmcblk1p2 failed fsck FAIL"
+        exit 2
+fi
+
+
+mkdir /tmp/core_sdcard_test
+
+mount -r /dev/mmcblk1p1  /tmp/core_sdcard_test/   
```

Comment:
probably should test that your mount was successful?  check the error code of this `mount` command?

Reply:
i.e.

```
if ! mount -r /dev/mmcblk1p1 /tmp/core_sdcard_test/; then echo "bad"; f
```